### PR TITLE
Add protobuf-matchers

### PR DESCRIPTION
**Pull request by mering**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (protobuf-matchers) have been updated in this PR. Please review the changes.

File: modules/protobuf-matchers/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "homepage": "https://github.com/inazarenko/protobuf-matchers",
+    "maintainers": [],
```
[the remark this answers is not in the document]
Comment (mering) [reply modules/protobuf-matchers/metadata.json]: Thanks!

**Comment by mering** [comment]
@fmeum could you please trigger the tests? Thanks!

File: modules/protobuf-matchers/metadata.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/inazarenko/protobuf-matchers",
+    "maintainers": [
+        {
+            "email": "igor.n.nazarenko@gmail.com",
+            "github": "inazarenko",
```
Comment (fmeum) [inline modules/protobuf-matchers/metadata.json:6]: @inazarenko Are you okay with being a maintainer of this module?

Comment (inazarenko) [reply modules/protobuf-matchers/metadata.json:6]: Yep. I will need to figure out how to update the registry when I make changes, but TBH this code is pretty static so there's not a lot of maintenance.

**Comment by mering** [comment]
Marking as draft to upstream the missing dependencies in https://github.com/inazarenko/protobuf-matchers/pull/7.

**Comment by mering** [comment]
@fmeum Tests are passing now.

**Pull request merged**